### PR TITLE
[NFR] [3.1.x] Tag prepend & append title feature

### PR DESCRIPTION
**Pull request by borisdelev**
Hello!

* Type: new feature
* Link to issue: https://github.com/phalcon/cphalcon/issues/12238

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Thanks

### CODE REVIEW

**Comment by sergeyklay** [comment]
@borisdelev Could you please rebase

**Review by sergeyklay — CHANGES_REQUESTED** [review]
The PSR check failed

**Comment by sergeyklay** [comment]
@borisdelev  Also could you please update CHANGELOG.md

**Comment by borisdelev** [comment]
Sorry i miss that :/ Thank you!

**Comment by sergeyklay** [comment]
Could you please fix appveyor.yml by changing
```yml
- if not exist vendor (php -r "readfile('https://getcomposer.org/installer');" | php)
```

to

```yml
- appveyor DownloadFile https://getcomposer.org/composer.phar
```

or something like this to fix Windows builds

**Review by sergeyklay — APPROVED** [review]

File: tests/unit/Tag/TagTitleTest.php [hunk 1]
```diff
@@ -110,6 +110,23 @@ function () {
...
                 expect(Tag::getTitle())->equals("<title>Main - Category - Title</title>" . PHP_EOL);
+                
+                Tag::resetInput();
+                
+                Tag::setTitle('Main');
+                Tag::setTitleSeparator(' - ');
+                Tag::appendTitle(['Category', 'Title']);
+                
+                expect(Tag::getTitle())->equals("<title>Main - Category - Title</title>" . PHP_EOL);
+
```
Comment (sergeyklay) [inline tests/unit/Tag/TagTitleTest.php]: a trailing space (tab) here

Comment (borisdelev) [inline tests/unit/Tag/TagTitleTest.php]: Please, excuse me (and sorry), but... cant get where exactly I must put space (tab) :/

**Comment by borisdelev** [comment]
I hope now all its fine. Thanks for your guide and kindness. 
Live long and prosper!

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**